packages/flutter/lib/src/material/bottom_sheet.dart

```diff
@@ -89,13 +92,27 @@ class BottomSheet extends StatefulWidget {
   /// Default is true.
   final bool enableDrag;
 
+  /// The bottom sheet's background color.
+  ///
+  /// Defines the bottom sheet's [Material.color].
+  ///
+  /// Defaults to [Colors.white].
+  final Color color;
+
   /// The z-coordinate at which to place this material relative to its parent.
   ///
   /// This controls the size of the shadow below the material.
   ///
   /// Defaults to 0. The value is non-negative.
   final double elevation;
 
+  /// The shape of the bottom sheet's [Material].
```

Comment:
Since the second line already mentions the `[Material]` (and that's a bit of an implementation detail), it might be better to just say that this defines the shape of the bottom sheet.

Reply:
Done